namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-intelligence-dev/resources/main.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  backend "s3" {
+  }
+}
+
+provider "aws" {
+  region = "eu-west-1"
```

Comment:
Is there any specific reason you want to have eu-west-1 (Ireland) as the default here, instead of eu-west-2 (London)?

Reply:
No, i just used a similar project as a base and it was using eu-west-1, happy to change

Reply:
👍 In general, it's best to use the cloud platform cli tool to create environments. That way, you can be sure you've provided all the information required, and have the standard defaults.

Reply:
👍